src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Email addins released
+category: Release Notes
+author: jericho
+---
+
+There are many scenarios where you need to send notifications during your build process, such as notifying colleagues that a build has passed/failed, notifying stakeholders that a package has been published to nuget, etc. One of the most popular ways to send such notification is, of course, email. That's why we recently released three new addins to send emails.
+
+__Cake.Email__
+
+The first addin is caled Cake.Email and allows you to send emails via your own SMTP server. 
+
+When invoking the `Email.SendEmail` alias, you must specify the name and email address of the sender (this is what the recipient will see in the `from` field when they receive your eamil). Typically, you would use your name and email address but you could use a generic address such as `notifications@yourcompany.com`), the name and email address of the recipient, the subject and content of the email, a boolean value indicating if the content should be sent as HTML or plain text and finally you must also specify the settings that are necessary to establish a connection to your SMTP server.
+
+Here's an example:
+
+```csharp
+#addin Cake.Email
+
+Task("SendEmail")
+    .Does(() =>
+{
+    try
+    {
+        var result = Email.Send(
+                senderName: "Bob Smith", 
+                senderAddress: "bob@example.com",
+                recipientName: "Jane Doe",
+                recipientAddress: "jane@example.com",
+                subject: "This is a test",
+                content: "<html><body>This is a test</body></html>",
+                sendAsHtml: true,
+                settings: new EmailSettings 
+                {
+                    SmtpHost = "smtp.gmail.com",
+                    Port = 587,
+                    EnableSsl = true,
+                    Username = "my_gmail_address@gmail.com",
+                    Password = "my_password"
+                }
+        );
+
+        if (result.Ok)
+        {
+            Information("Email succcessfully sent");
+        }
+        else
+        {
+            Error("Failed to send email: {0}", result.Error);
+        }
+    }
+    catch(Exception ex)
+    {
+        Error("{0}", ex);
+    }
+});
+```
+
+
+__Cake.SendGrid__
```

Comment:
Can you make this a link to the NuGet Package?

Also, can you add a link within the paragraph to the source repository?